docs/anonymised-data.md

```diff
@@ -0,0 +1,37 @@
+# RCA - Anonymising data
+
+When pulling data from any hosted instance, take a cautious approach about whether you need full details of potentially personally-identifying, confidential or sensitive data.
+
+## General principles:
+
+- pull data from staging rather than production servers, if this is good enough for your needs
+- if it is necessary to pull data from production, e.g. for troubleshooting, consider whether anonymising personal data is possible and compatible with your needs
+- if it is necessary to pull non-anonymised data from production, consider destroying this copy of the data as soon as you no longer need it
```

Comment:
Extremely small nitpick for docs consistency.

```suggestion
- Pull data from staging rather than production servers, if this is good enough for your needs
- If it is necessary to pull data from production, e.g. for troubleshooting, consider whether anonymising personal data is possible and compatible with your needs
- If it is necessary to pull non-anonymised data from production, consider destroying this copy of the data as soon as you no longer need it
```